### DISCUSSION TITLE
Eclipse Installer build frequency

### DISCUSSION THREAD

**Discussion by kolovos** [eclipse-oomph/oomph#29] [Q&A]
How frequently do the nightly builds of the Eclipse Installer linked from https://wiki.eclipse.org/Eclipse_Installer actually get built? I'm trying to install the latest interim version of Epsilon and the nightly version of the [Mac OS 64 Bit installer](http://www.eclipse.org/downloads/download.php?file=/oomph/products/latest/eclipse-inst-mac64.dmg) seems to be using an older version of Epsilon's [epsilonUse.setup](https://github.com/eclipse/epsilon/blob/main/releng/org.eclipse.epsilon.releng/epsilonUse.setup), which contains a feature (org.eclipse.epsilon.eugenia.feature) that no longer exists after [this commit](https://github.com/eclipse/epsilon/commit/273ae8e27c8afa7c947526d043ea18ad2b9f6845).

**Comment by merks** [comment]
The installer does not contain the setups.  This job builds a zip

https://ci.eclipse.org/oomph/job/setup-archiver/

The installer downloads that zip. Sometimes it may take a while for the www.eclipse.org server serves the updated version.

**Reply by merks** [reply]
A build that promotes changes is kept and I does see one with changes from you.

**Reply by kolovos** [reply]
Thanks Ed. It seems that we forgot to update the URLs of Epsilon's setup files when we moved to GitHub but I've just submitted a [pull request](https://github.com/eclipse-oomph/oomph/pull/30) that fixes this.

**Reply by merks** [reply]
This picked up the changes

https://ci.eclipse.org/oomph/job/setup-archiver/65943/console
